frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -644,6 +644,10 @@ export const surveyLogic = kea<surveyLogicType>([
 
                 const response = await api.query(query)
                 actions.setBaseStatsResults(response.results as SurveyBaseStatsResult)
+                const numberOfSurveySentEvents = response.results?.find(
+                    (result) => result[0] === SurveyEventName.SENT
+                )?.[1]
```

Comment:
what if `result` is empty or the find returns empty or a single item.
the `[0]` and `[1]` will throw

Reply:
if it didn't have the `?` then it would throw